jax/_src/core.py

```diff
@@ -462,6 +469,14 @@ def __init__(self, val, aval):
 
   __hash__ = None  # type: ignore
 
+  def pretty_print(self, context: JaxprPpContext, *, print_dtype: bool = True):
```

Comment:
Why take `context` as an argument? `pretty_print` does not mean that this is for jaxpr right?

And given that it's not used, can we remove it? We can add it later if we want.

Reply:
Ohh sorry I missed the Var usage. Cool LGTM

Reply:
No problem - thank you!!

Reply:
Sorry for the back and forth 😢